workflow/hpxml_inputs.json

```diff
@@ -1689,6 +1694,15 @@
     "heat_pump_heating_autosizing_factor": 1.4,
     "heat_pump_backup_heating_autosizing_factor": 0.9
   },
+  "sample_files/base-hvac-air-to-air-heat-pump-1-speed-autosize-limit.xml": {
```

Comment:
Rather than adding a whole bunch of extra test files and simulations for the autosizing limits, how about just incorporating the new input into the 4 existing `base-hvac-foo-autosize-factor.xml` files? Look at what autosized capacities they are getting and then choose something a bit less to demonstrate they are limiting the capacity.

Reply:
We are therefore ok not checking results when specifying autosizing factors?

Reply:
Why wouldn't you be able to check results? I'm not following.

Reply:
I just mean that the autosize factor values become arbitrary then, right?

Reply:
No...

If autosized value = 30k Btuh, and you set the autosize factor to be 1.5 and the autosize limit to be 40k Btuh, you should end up at 40k Btuh. Both the factor and limit applied.

Reply:
We're not testing simulation at the 1.5x autosized value. That's all I meant.

Reply:
They are almost all above 1. And if you want to change them to all be above 1, that's fine.